docs/source/asr/speech_classification/results.rst

```diff
@@ -0,0 +1,77 @@
+Checkpoints
+===========
+
+There are two main ways to load pretrained checkpoints in NeMo:
+
+* Using the :code:`restore_from()` method to load a local checkpoint file (`.nemo`), or
+* Using the :code:`from_pretrained()` method to download and set up a checkpoint from NGC.
+
+See the following sections for instructions and examples for each.
+
+Note that these instructions are for loading fully trained checkpoints for evaluation or fine-tuning.
+For resuming an unfinished training experiment, please use the experiment manager to do so by setting the
+``resume_if_exists`` flag to True.
+
+Loading Local Checkpoints
+-------------------------
+
+NeMo will automatically save checkpoints of a model you are training in a `.nemo` format.
+You can also manually save your models at any point using :code:`model.save_to(<checkpoint_path>.nemo)`.
+
+If you have a local ``.nemo`` checkpoint that you'd like to load, simply use the :code:`restore_from()` method:
+
+.. code-block:: python
+
+  import nemo.collections.asr as nemo_asr
+  model = nemo_asr.models.<MODEL_BASE_CLASS>.restore_from(restore_path="<path/to/checkpoint/file.nemo>")
+
+Where the model base class is the ASR model class of the original checkpoint, or the general `ASRModel` class.
+
+NGC Pretrained Checkpoints
+--------------------------
+
+The Speech Classification collection has checkpoints of several models trained on various datasets for a variety of tasks.
+These checkpoints are obtainable via NGC `NeMo Automatic Speech Recognition collection <https://ngc.nvidia.com/catalog/models/nvidia:nemospeechmodels>`_.
+The model cards on NGC contain more information about each of the checkpoints available.
+
+The tables below list the Speech Classification models available from NGC, and the models can be accessed via the
+:code:`from_pretrained()` method inside the ASR Model class.
+
+In general, you can load any of these models with code in the following format.
+
+.. code-block:: python
+
+  import nemo.collections.asr as nemo_asr
+  model = nemo_asr.models.EncDecClassificationModel.from_pretrained(model_name="<MODEL_NAME>")
+
+Where the model name is the value under "Model Name" entry in the tables below.
+
+For example, to load the MatchboxNet3x2x64_v1 model for speech command detection, run:
```

Comment:
No examples - ASR and rest dont have so much duplication. Revert this para

Reply:
ASR has "For example, to load the base English QuartzNet model for speech recognition, run: ..." 
Do we need to delete it?

Reply:
Hmm seems fine to keep then.